workflow/Snakefile

```diff
@@ -151,11 +152,14 @@ rule call_neighborhoods:
 			--ubiquitously_expressed_genes {params.ubiquitous_genes} \
 			--qnorm {params.qnorm} \
 			--H3K27ac {params.H3K27ac}
+		# This command is to make sure the command above has finished running. If the job does not finish within the slurm job time limit, snakemake would still think the job has completed because the output files exist. 
```

Comment:
I thought if the output files (enh and genelist) exist, that warrants a successful run. Are you saying the output files can be incomplete? Or are there other output files we need to look for?

Reply:
The output files could be incomplete

Reply:
ah i see. I guess this works for now but I think the right solution could be to create a temporary file and then rename it to the right filename once everything has been written through

Reply:
I see. I can make that change?

Reply:
if you can, that would be great!